src/debug/debug.cpp

```diff
@@ -859,7 +859,9 @@ static void DrawCode(void) {
 		if (showExtend) res = AnalyzeInstruction(dline, saveSel);
 		// Spacepad it up to 28 characters
 		size_t dline_len = strlen(dline);
-		if(dline_len < 28) for (c = dline_len; c < 28;c++) dline[c] = ' '; dline[28] = 0;
+		for (c = dline_len; c < 28; c++)
+			dline[c] = ' ';
+		dline[28] = 0;
```

Comment:
Very nice! (and safe; confirmed that `c` is only ever assigned before use further up, and not used below - so there are no side-effects of using assigning `c = dline_len` straight-away.